package.json

```diff
@@ -169,6 +169,7 @@
     "jsdom": "^22.0.0",
     "lint-staged": "^13.0.3",
     "prop-types": "^15.8.1",
+    "react-devtools": "^4.27.7",
```

Comment:
You can use the React DevTools browser extension without adding an additional package to the project. The package installation is helpful for debugging non-browser-based (e.g. React Native, mobile browser, or embedded webview) or Safari.

Reply:
Got it. Removed the package.